### DISCUSSION TITLE
Linux Icon Poll

### DISCUSSION THREAD

**Discussion by daiyam** [VSCodium/vscodium#1477] [Polls]
Vote for VSCodium's app icon on Linux!

We would like your feedback about the icon on Linux!

The vote ends the 20th April at midnight.

|                                                       `circle1`                                                       | `macos1`                                                                                                             | `nobg`                                                                                                             | `rect1`                                                                                                             |
| :-------------------------------------------------------------------------------------------------------------------: | -------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------- |
| <img src="https://raw.githubusercontent.com/VSCodium/icons/main/icons/linux/circle1/blue1/paulo22s.png" width="128"/> | <img src="https://raw.githubusercontent.com/VSCodium/icons/main/icons/linux/macos1/blue1/paulo22s.png" width="128"/> | <img src="https://raw.githubusercontent.com/VSCodium/icons/main/icons/linux/nobg/blue1/paulo22s.png" width="128"/> | <img src="https://raw.githubusercontent.com/VSCodium/icons/main/icons/linux/rect1/blue1/paulo22s.png" width="128"/> |

If you have suggestions, please add your background template to [icons](https://github.com/VSCodium/icons). We might run another vote before the next version/end of month.

**Comment by DanielProg39** [comment]
I don't know... The good old round icon looks good, but the new transparent one isn't bad either. I like both and for me there's not much difference. Rect1 also looks appealing to me

**Reply by geekley** [reply]
Here's how they all look in KDE Plasma's default dark and white themes, at 32px:
![circle1](https://user-images.githubusercontent.com/38332912/231890990-953387fa-ae36-46c5-8aca-eb15a9cfd8e3.png) ![macos1](https://user-images.githubusercontent.com/38332912/231890996-02365317-6cde-496a-8912-c01034511080.png) ![nobg](https://user-images.githubusercontent.com/38332912/231890998-47c6c98d-365a-4512-adfe-5a62bee38f36.png) ![rect1](https://user-images.githubusercontent.com/38332912/231890999-1076d300-abf9-4eae-b294-e614a26c44b6.png) ![w-circle](https://user-images.githubusercontent.com/38332912/231896681-660d4ce5-e36c-4251-b623-399a8805aafb.png) ![w-macos1](https://user-images.githubusercontent.com/38332912/231896687-b902d8cd-067e-461a-a9b0-46fef8215f9f.png) ![w-nobg](https://user-images.githubusercontent.com/38332912/231896688-ab75dcc7-1df1-4212-9877-d3e53de52d1b.png) ![w-rect](https://user-images.githubusercontent.com/38332912/231896691-0bf0b17a-6a81-4a47-85cb-924aadcf78bf.png)
On default Gnome, `nobg` might not be an issue, but any theme/DE with similar blue bg/accent color makes it have bad contrast. This isn't a problem with other blue icons because they at least have a second color, even if it's minor.
It's worse in Unity DE, which **purposefully** uses a BG with the **exact same** main color of the icon.
![nobg-Unity](https://user-images.githubusercontent.com/5273753/229796968-ec7a1733-20f9-4ad3-84a0-c6bbbff315ff.png) ![circle2-Unity](https://user-images.githubusercontent.com/5273753/232202388-7b0390d1-67ba-4152-8fc9-64806fbc7e08.png)
This wouldn't happen if `nobg` had at least some border (though I'll admit it doesn't fit too well with other app icons):
![border-Unity](https://user-images.githubusercontent.com/5273753/232201229-e6b02eac-ba82-45e2-9e1d-cb52be183dca.png)

It would be nice if people who use other distros/DEs could post a similar comparison here.

**Reply by DanielProg39** [reply]
Here's how the current icon looks like on my desktop
![IMG_20230414_101454_450](https://user-images.githubusercontent.com/63293432/231971194-3dd08b56-ff31-4949-b99b-c1695a5bfde2.jpg)
![Снимок экрана от 2023-04-14 10-11-04](https://user-images.githubusercontent.com/63293432/231971202-8e6291db-4725-42fc-bf37-576ffa9edcef.png)

**Comment by daiyam** [comment]
@geekley You are right, they are small. I will fix them...

![Screen Shot 2023-04-13 at 10 02 28 AM](https://user-images.githubusercontent.com/587742/231694368-5d4cdf47-971c-40fd-935f-9fa6eb975530.png)

**Reply by daiyam** [reply]
The icons have been updated https://github.com/VSCodium/icons/commit/d926d66336364dee542a2867727f57cabe145fb1

![Screen Shot 2023-04-13 at 11 13 28 AM](https://user-images.githubusercontent.com/587742/231713143-f0c3493c-837f-4e62-82ee-4894ee75c9e7.png)

**Comment by geekley** [comment]
Guys, it's important that you vote anything except `nobg`, here's why:
KDE
![](https://user-images.githubusercontent.com/38332912/229219139-0fec15d8-21bb-482b-b472-e5a77c77dffa.png)
Unity
![](https://user-images.githubusercontent.com/5273753/229796968-ec7a1733-20f9-4ad3-84a0-c6bbbff315ff.png)
It needs to have 2 distinct colors at a minimum.

**Reply by daiyam** [reply]
Just for curiosity, have you tried my [fork](https://github.com/zokugun/MrCode)? How does the icon look on those systems?

**Reply by geekley** [reply]
I haven't, but replacing just the icon on KDE it would be like this (taskbar, menu):
![MrCode on KDE taskbar](https://user-images.githubusercontent.com/38332912/231700980-6cac48d3-0525-4ec4-a508-d08bbd80b575.png) ![MrCode on KDE Menu](https://user-images.githubusercontent.com/38332912/231700987-3dc6852f-b937-42f6-884a-8e83bbe785cf.png)
If you're asking for my opinion, using a blue/black/whatever filling inside would avoid a similar problem if one happens to use a green theme (though it might be unusual?) or uses Unity DE (I don't, so I can't test it, but from what I remember, it would put a green BG because it detects and matches the icon's "major" color). Or you could just have a second color anywhere (e.g. hexagon stroke different than mountains -- note how Thunderbird, Kate and Dolphin aren't an issue with blue because there's a bit of white too); in fact I'm not sold on the green-brownish part at the top, so maybe using black/white for the hex and keep green only in mountain? idk

**Reply by daiyam** [reply]
Thank you for the feedback!

**Reply by daiyam** [reply]
There must be a way on Linux to able to a custom icon for an app, no?

**Reply by mndscp** [reply]
> There must be a way on Linux to able to a custom icon for an app, no?

Yes, there is: https://github.com/VSCodium/icons#how-to-use-them
For KDE at least you can simply edit the line that starts with `Icon` in `/usr/share/applications/codium.desktop`.

**Reply by itkfm** [reply]
![image](https://user-images.githubusercontent.com/53079668/231767393-e679546d-523d-4c99-9a68-64a147270e66.png)
the invisibly thin nobg version kinda “drowns” in the taskbar

**Comment by subcidr** [comment]
The benefit of the `nobg` is that it will fit better on different kinds of backgrounds. Otherwise if people want a transparent background they need to remove the bg themselves... So my vote goes to nobg 💯

**Reply by geekley** [reply]
Unless background is blue, in which case it's practically invisible, since it has no outline, no border or any second color.
It's already bad enough for me, so it must be even worse for people which need increased contrast accessibility ...
And it's not like blue as an accent/BG color is at all uncommon -- on KDE dark theme blue accent color is the default.

**Reply by subcidr** [reply]
Hmm you're quite right about that, didn't see your comment above before now. `circle1` would be better in that case.

**Reply by masakk1** [reply]
how many people have a blue taskbar compared to a white/black one though?

**Reply by geekley** [reply]
It's not just the color of whole taskbar, it's the BG color where the icon appears, which can be blue when app is active and/or hovered. So, at least any KDE users on default dark theme (though contrast is not THAT bad) + any Unity DE (the one Ubuntu used before switching to Gnome) user (where it's pretty much invisible, like shown in the other post above). I couldn't find stats on % of users for each Linux DE, so it's difficult to answer that question, but not all Linux users are Ubuntu and not all Ubuntu users are on Gnome.

But honestly, the mere fact users *can* customize a theme arbitrarily should be enough for having a guideline where "icons should not be composed of only 1 single color", period. I'm not discussing preference here (I don't care, I'll just make it the way I want anyways), I'm discussing accessibility for others.

**Comment by MarkJeronimus** [comment]
I'd prefer the circle if it didn't have the shadow that makes it look pseudo-3D

**Reply by geekley** [reply]
Then you might wanna look at the one I had made there, with a different shadow: https://github.com/VSCodium/vscodium/discussions/1218#discussioncomment-5586517
Though, tbh I personally prefer this version; my taskbar is at 32px, the shadow/thickness is not even noticeable.
And I kinda like that it's just a bit like a button.

**Reply by mndscp** [reply]
Actually not bad, thank you. I also like this version without the shadow better.
Thread is full of "My taste is more like ..." :P

**Comment by brodysmith8** [comment]
What if nobg, but with a white outline? That way, there is built-in contrast to accommodate both lighter backgrounds, and darker backrounds. 

Will the macos1 variant be kept for the macOS release, or will macOS also get the Linux icon? macos1 fits in well with the other rounded-square icons in the bottom dock.

**Reply by daiyam** [reply]
> white outline

I tried but it was weird.

 > Will the macos1 variant be kept for the macOS release

Each OS will have their own variant, the logo and color are still the same. macos1 will stay on macOS.

**Reply by geekley** [reply]
You're talking about this, right?
<img src="https://raw.githubusercontent.com/VSCodium/vscodium/99da974ce5f1126a74c8bfedc07e3103a61c5272/icons/stable/codium_cnl_w80_b8.svg" height=64/> <img src="https://raw.githubusercontent.com/VSCodium/vscodium/99da974ce5f1126a74c8bfedc07e3103a61c5272/icons/stable/codium_cnl_w80_b8.svg" height=32/> <img src="https://raw.githubusercontent.com/VSCodium/vscodium/99da974ce5f1126a74c8bfedc07e3103a61c5272/icons/stable/codium_cnl_w80_b8.svg" height=16/>
The idea behind it is not bad, at least it's better than `nobg`, since it solves the contrast issue, so maybe there could be some middle-ground (seeing as `nobg` is likely to win :disappointed: )? Perhaps if stroke is much thinner (e.g 1/4 of current size) or has a discreet white glow instead of a thick outline? Like, something almost the same as `nobg`, but with just enough contrast to minimize the issue.

**Reply by geekley** [reply]
Experimenting with this. Using a similar colored border seems to be the way to keep it true to the idea of `nobg`, but improve the contrast. I went with a-bit-darker-blue stroke, but it *might* also work with a-bit-lighter-blue stroke.
On dark theme, the difference is negligible at 32px and below, but contrast against blue bg improves a lot. It only looks slightly worse on white theme at 16px, but at this size it's difficult with this design; this is the best I could do (maybe a bit bigger with less empty space around it could improve this?).
![codium-stable-border](https://user-images.githubusercontent.com/38332912/232164407-98fa1978-0a8c-4f67-947a-1bc0de98f0d8.png) <img src="https://user-images.githubusercontent.com/38332912/232164407-98fa1978-0a8c-4f67-947a-1bc0de98f0d8.png" height=64/> <img src="https://user-images.githubusercontent.com/38332912/232164407-98fa1978-0a8c-4f67-947a-1bc0de98f0d8.png" height=32/> <img src="https://user-images.githubusercontent.com/38332912/232164407-98fa1978-0a8c-4f67-947a-1bc0de98f0d8.png" height=16/> ![codium-stable-border-bg-dark](https://user-images.githubusercontent.com/38332912/232166604-8531cb24-d56d-4274-ae0a-6a772f5ac160.png) <img src="https://user-images.githubusercontent.com/38332912/232166604-8531cb24-d56d-4274-ae0a-6a772f5ac160.png" height=64/> <img src="https://user-images.githubusercontent.com/38332912/232166604-8531cb24-d56d-4274-ae0a-6a772f5ac160.png" height=32/> <img src="https://user-images.githubusercontent.com/38332912/232166604-8531cb24-d56d-4274-ae0a-6a772f5ac160.png" height=16/>
![codium-stable-border-bg](https://user-images.githubusercontent.com/38332912/232164401-6cd7fa92-f133-474a-9c0a-eafd95523f57.png) <img src="https://user-images.githubusercontent.com/38332912/232164401-6cd7fa92-f133-474a-9c0a-eafd95523f57.png" height=64/> <img src="https://user-images.githubusercontent.com/38332912/232164401-6cd7fa92-f133-474a-9c0a-eafd95523f57.png" height=32/> <img src="https://user-images.githubusercontent.com/38332912/232164401-6cd7fa92-f133-474a-9c0a-eafd95523f57.png" height=16/> ![codium-stable-border-bg-light](https://user-images.githubusercontent.com/38332912/232166607-1d2c9e38-5860-481a-bcff-42e3596ddc88.png) <img src="https://user-images.githubusercontent.com/38332912/232166607-1d2c9e38-5860-481a-bcff-42e3596ddc88.png" height=64/> <img src="https://user-images.githubusercontent.com/38332912/232166607-1d2c9e38-5860-481a-bcff-42e3596ddc88.png" height=32/> <img src="https://user-images.githubusercontent.com/38332912/232166607-1d2c9e38-5860-481a-bcff-42e3596ddc88.png" height=16/>
<details><summary>SVG</summary>

```svg
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   version="1.1"
   x="0"
   y="0"
   width="128"
   height="128"
   viewBox="0 0 128 128"
   id="svg"
   sodipodi:docname="codium-stable-border.svg"
   inkscape:version="1.1.2 (0a00cf5339, 2022-02-04)"
   inkscape:export-filename="codium-stable-border.png"
   inkscape:export-xdpi="96"
   inkscape:export-ydpi="96"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <sodipodi:namedview
     id="namedview17"
     pagecolor="#3678c9"
     bordercolor="#ffffff"
     borderopacity="1"
     inkscape:pageshadow="0"
     inkscape:pageopacity="0"
     inkscape:pagecheckerboard="1"
     showgrid="true"
     inkscape:current-layer="path6008">
    <inkscape:grid
       type="xygrid"
       id="grid826"
       spacingx="4"
       spacingy="4"
       empspacing="4"
       empcolor="#9f9f9f"
       empopacity="0.125"
       color="#9f9f9f"
       opacity="0.0625" />
  </sodipodi:namedview>
  <defs
     id="defs7">
    <linearGradient
       id="Gradient_1"
       gradientUnits="userSpaceOnUse"
       x1="209.998"
       y1="224.04201"
       x2="287.332"
       y2="298.65799"
       gradientTransform="matrix(1.28,0,0,1.28,-255.99944,-272.64508)">
      <stop
         offset="0"
         stop-color="#62A0EA"
         id="stop2" />
      <stop
         offset="1"
         stop-color="#1A5FB4"
         id="stop4" />
    </linearGradient>
  </defs>
  <g
     id="Layer_1">
    <g
       id="path6008">
      <path
         d="m 62.218799,15.123239 c -2.92224,1.8752 -3.76576,5.75872 -1.8816,8.672 c 10.35136,16.03072 12.79488,28.1216 11.73504,38.71744 c -4.28544,20.29184 -13.59872,24.40832 -21.73696,24.40832 c -7.94624,0 -5.81888,-13.07776 0.15488,-16.90496 c 3.5648,-2.22592 8.13184,-3.6736 11.63776,-3.6736 c 3.47264,0 6.28864,-2.80576 6.28864,-6.26944 c 0,-3.46368 -2.816,-6.27072 -6.28864,-6.27072 c -4.09472,0 -8.11008,0.86016 -11.8592,2.32448 c 0.76544,-3.61344 1.04832,-7.52 0.05888,-11.67104 c -1.504,-6.304 -5.83168,-12.32512 -13.34528,-18.16192 c -1.31584,-1.024 -2.98496,-1.48352 -4.64128,-1.27872 c -1.65504,0.2048 -3.1616,1.05728 -4.18688,2.36928 c -2.1312,2.73408 -1.6384,6.6752 1.10464,8.80128 c 6.12096,4.7552 8.14208,8.27392 8.83328,11.17184 c 0.6912,2.89664 0.13568,6.00448 -1.25312,10.27456 c -1.77408,5.75488 -3.83488,10.89152 -4.73856,15.81824 c -0.44416,2.4256 -0.48128,5.06368 -0.59648,7.17184 c -4.40832,-4.29568 -6.13376,-9.96864 -6.13376,-18.25792 c -0.0013,-3.46368 -2.81728,-6.272 -6.28992,-6.27072 c -3.47008,0.0038 -6.28224,2.8096 -6.28224,6.27072 c 0,11.32032 3.31008,22.0928 12.18304,29.28 c 8.032,7.65184 28.45824,4.82432 28.45824,17.108481 c 0,3.46752 5.0752,5.15456 8.54656,5.15456 c 3.56224,0 8.04352,-2.3808 8.04352,-5.15456 c 0,-13.937921 14.68928,-22.410241 38.871041,-22.373121 c 3.47392,0.0051 6.29248,-2.79936 6.2976,-6.26304 c 0.005,-3.46368 -2.80448,-6.27712 -6.27712,-6.28352 c -1.65376,-0.0013 -3.26656,0.05504 -4.85504,0.14848 c 2.70336,-6.35776 3.90272,-13.35808 3.6544,-20.91904 c -0.1152,-3.4624 -3.0208,-6.176 -6.49088,-6.06336 c -3.475201,0.11264 -6.196481,3.0144 -6.082561,6.47808 c 0.32768,9.9008 -0.04352,18.74048 -7.41888,23.40096 c -2.09792,1.32352 -4.53504,2.47296 -6.848,2.47296 c 1.79584,-4.8832 3.15136,-10.03648 3.7056,-15.59168 c 0.35456,-3.5456 0.39296,-7.7568 -0.01152,-11.02848 c -0.62848,-5.0688 -1.38624,-10.81728 0.53888,-15.14752 c 1.728,-3.72992 5.60128,-5.29152 11.28704,-5.29152 c 3.47008,-0.0026 6.282241,-2.8096 6.283521,-6.26944 c 10e-4,-3.46496 -2.810881,-6.27456 -6.283521,-6.27712 c -8.45056,0 -14.85568,4.44672 -18.48576,9.82016 c -1.89952,-4.05504 -4.2112,-8.23424 -7.00416,-12.55936 c -0.9024,-1.39776 -2.32448,-2.3808 -3.95264,-2.73408 c -0.80768,-0.17408 -1.64096,-0.18944 -2.45376,-0.0448 c -0.8128,0.1472 -1.58976,0.45056 -2.2848,0.896 z"
         fill="url(#Gradient_1)"
         id="path9"
         stroke="#111"
         stroke-width="12"
         style="fill:url(#Gradient_1);stroke-linejoin:round;stroke-opacity:1;paint-order:markers stroke fill;stroke:#1b3c64;stroke-width:8;stroke-miterlimit:4;stroke-dasharray:none" />
    </g>
  </g>
</svg>

```
</details>

Anyways, if `nobg` wins, maybe you could ask people if they would be okay with something like this instead, to solve the contrast issue against blue BGs.

**Reply by daiyam** [reply]
We need to check the black background with the blue overlay of KDE  or Unity.

**Reply by daiyam** [reply]
The next vote should include preview of different backgrounds with/without overlays.
If there is any tools that can automate that, it would be great!

**Reply by geekley** [reply]
> If there is any tools that can automate that, it would be great!

I don't know of any tools, but for KDE, at the very least you can pick colors from the screenshots and show those (BG is not dependent on the icon, unlike Unity). Or you could take/edit screenshots with invisible icons and use that template to generate, placing the icon on top of it. You'd just need a reference full-square icon to place it correctly.
I took the screenshots manually using Spectacle; while it's open, it remembers the screen region you use to crop so you can take several pics in the same place. But switching icons manually is a pain:
- edit `Icon` in ~/.local/share/applications/codium.desktop
- to refresh icon, delete the .desktop file; wait a little; <kbd>Ctrl+Z</kbd>; wait a little
- take screenshot
- repeat for other icons ...

> We need to check the black background with the blue overlay of KDE or Unity.

I expect in Unity it may make a significant difference, but I can't test it; @MarkJeronimus
In KDE there's a slight improvement, but not a huge difference because the issue is not as bad as in Unity:
![screenshot-blue](https://user-images.githubusercontent.com/38332912/232199907-0fd6500c-04d0-4270-aa6e-c40dd7e79b53.png) ![screenshot-blue-light](https://user-images.githubusercontent.com/38332912/232200148-464eb53f-ffad-4587-b475-e928e281d168.png)
(blue overlay color in KDE dark theme is `#326784`, light theme: `#9ED2ED`)
I'll also admit, in light theme it looks a bit out-of-place compared to the other icons :disappointed:

**Reply by MarkJeronimus** [reply]
Unity DE 7.5 (7.6 bundled with the "Ubuntu Unity" distro is unavailable for me):
![image](https://user-images.githubusercontent.com/5273753/232200955-38eab0fe-adc5-4a31-98e5-710fca3569ee.png)
blue outline (the coral seems smaller, but I'm on a different PC now which might behave differently. The icon size is set to 30):
![image](https://user-images.githubusercontent.com/5273753/232201229-e6b02eac-ba82-45e2-9e1d-cb52be183dca.png)

Deleting the .desktop file and waiting a bit didn't work. It only changed the launcher icon, not he pin/program icon. I had to delete the icon in /usr/share/pixmaps instead and place it back.

[Edit] the round one which I asked about in a thread above still has a shadow effect:
![image](https://user-images.githubusercontent.com/5273753/232202388-7b0390d1-67ba-4152-8fc9-64806fbc7e08.png)

**Reply by geekley** [reply]
No, that's right, I accidentally made it smaller.
> Deleting the .desktop file and waiting a bit didn't work.

Don't know if it's the same issue, but it's easier if you use different icon names/paths, so it doesn't use the cached version. For testing, you can use absolute path instead of pixmaps, I think.

**Reply by daiyam** [reply]
@geekley I've added your variant to https://vscodium-icons.netlify.app/

How did you get the border color?

**Reply by geekley** [reply]
I just "arbitrarily" used color picker on where it seemed like the middle of the gradient, then reduced HSL's luminosity value to make it darker.

**Reply by daiyam** [reply]
I've slightly changed the color of the border to keep the same hue and applied the diff to the others colors... 

Should we just go for the `nobg/border` or do another poll against `circle1`?

@geekley @GitMensch @paulcarroty @mndscp What do you think?

The new icon can be seen at https://vscodium-icons.netlify.app/

**Reply by geekley** [reply]
Well I voted circle, so take what I say with a grain of salt; I don't wanna force my opinion on anyone.
However, nobg didn't win by absolute majority (>50%). Since it's the one most different from the others, it might be plausible to assume rect/macos votes could have gone to any of the 3 which was against nobg.

It's up to you to decide, but if you want to include circle, you can make it so it needs 50% majority to win, otherwise, whichever of nobg or border has more votes wins even if circle gets more votes (as if they belong to the same "party", with joint votes against circle). This way we can understand what rect/macos votes wanted without being unfair to this poll's winner, IMO.

**Reply by GitMensch** [reply]
I'm no artist so I can only share an opinion - I only do this because of the explicit request...
for me, the circle looks "clear" and identical on all backgrounds, the version with the border looks "hard" to me (and not well recognizable in small sizes) instead of the "open" non-border version

I will go with any default, just did hope we don't get the potty as default ;-)

**Reply by daiyam** [reply]
OK, let's go for the final polls between those 2 for both Linux and Windows!

**Reply by geekley** [reply]
Which two?

**Reply by daiyam** [reply]
`nobg/border` (the one you've made) vs `circle1 `

**Reply by geekley** [reply]
Ah, OK. But just to confirm, did you understand my suggestion? There would be 3 options, and votes for `nobg` (winner here) and votes for `border` (the one I made) would be added together. In 1 poll you can solve both `nobg, border` vs `circle` and `nobg` vs `border` (if circle <= `50%`).

**Reply by daiyam** [reply]
I was planning to do 2 polls of 5 days (best 2 and best 1) but I find that `nobg/border` is superior to `nobg` and it was kinda something I wanted. And to do 2 polls is too much... 2 votes in 1 poll/question will be confusing, a poll can only ask 1 question...

**Reply by geekley** [reply]
Ah I see, so by `nobg/border` you mean what I just called `border`. So it would be just `border` vs `circle`? Well, you decide.
I was just afraid that a significant amount of people may want `nobg`, but not like `border`. Because while `border` is surely superior wrt solving the contrast issue, I also understand that its style is different from all other apps (at least in Linux). So I have no idea if it may have a strong rejection, and if people may feel like it doesn't fit with the other apps (I myself am not too sure :sweat_smile:). It would be good to have some feedback on this, what do you guys think?

IMO, I don't think it would be confusing to say "since circle was second last poll, it can only win now if it has >50% votes".
But I see what you mean, and I'm fine with whatever you decide :grin:

**Comment by evur** [comment]
Please don't use shadows on the icons.

**Comment by daiyam** [comment]
I've added a previewer in https://github.com/VSCodium/icons/tree/main/previewer and deployed it at https://vscodium-icons.netlify.app/

**Comment by daiyam** [comment]
Next stage at [Semi-Finale Poll #1488](https://github.com/VSCodium/vscodium/discussions/1488)
